modules/statistics/RTPStatsCollector.js

```diff
@@ -2,6 +2,7 @@ import browser from '../browser';
 import { browsers } from 'js-utils';
```

Comment:
Just had a thought. Tell me what you think: put the stats formatting into a separate module. That separates the collecting from the formatting. The separate module could then conditionally load the right formatter for rtpstatscollector to use (or the collector conditionally loads the right formatter).

Reply:
What do you mean by formatter? So currently what has been collected is been formatted in _processAndEmitReport which is common, no matter of the type of the stats...

Reply:
I mean separating the collecting and the formatting logic. Conceptually they seem like two different concepts and the collecting logic doesn't necessarily need to know about formatting. That way the collector can use an arbitrary formatter, thereby hiding that there are two implementations for formatting.

Reply:
But I now get the feeling, since you mentioned instance variable somewhere else, that the collector and formatting may not have the clean separation already that could allow the collector to take and use an arbitrary formatter.